src/main/java/com/vmtapp/enterprise/service/TicketService.java

```diff
@@ -93,4 +101,19 @@ public Ticket save(Ticket ticket) {
     public List<Ticket> fetchAll(){
         return ticketSQLDAO.fetchAll();
     }
+
+    @Override
+    public void saveImage(MultipartFile imageFile) throws IOException {
+        String folder = "/It4045photo/";
+        // make folder if it does not exist
+        File directory = new File(folder);
+        if (! directory.exists()){
+            directory.mkdir();
+        }
```

Comment:
other then what was part of the class room video, this my attempt to make a directory folder if it does not exist. If I more knowledge of the database that we using I try to it there, but for not they should be a good starting point.